apps/site/src/views/Landing/components/ApplyButton/ApplyButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "bootstrap-utils" as utils;
+
+.applyButtonContainer {
+	position: relative;
+}
+
+.applyButton {
+	position: relative;
+	transform: skew(-30deg);
+	height: 80px;
+	width: 250px;
+	margin: 0px;
+	border: none;
+	background-color: utils.$yellow;
```

Comment:
Issue: the highlighter yellow is slightly transparent (70% opacity) in the provided design